types/mui-datatables/index.d.ts

```diff
@@ -461,5 +541,9 @@ export const TableSelectCell: React.Component<MUIDataTableSelectCell>;
 export const TableToolbar: React.Component<MUIDataTableToolbar>;
 export const TableToolbarSelect: React.Component<MUIDataTableToolbarSelect>;
 export const TableViewCol: React.Component<MUIDataTableViewCol>;
+export const DebounceTableSearch: React.Component<DebouncedMUIDataTableSearch>;
+
+// Plugins
+export function debounceSearchRender(debounceWait: number): MUIDataTableOptions['customSearchRender'];
```

Comment:
isn't the `debounceWait` optional (defaults to 200ms)?
https://github.com/gregnb/mui-datatables/blob/468c9c7e339501a786caf7b6afbea2c4f1f63a6f/src/plug-ins/DebounceSearchRender.js#L103
```suggestion
export function debounceSearchRender(debounceWait?: number): MUIDataTableOptions['customSearchRender'];
```

Reply:
yeah good catch